railties/lib/rails/generators/rails/app/templates/github/dependabot.yml

```diff
@@ -5,3 +5,8 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 10
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
I used `daily` to keep it consistent with the config for `bundler`. Personally speaking, I would change both to `weekly`, since in my experience `daily` would be a bit frustrating.